docs/send-data/kubernetes/collecting-logs.md

```diff
@@ -504,6 +504,128 @@ sumologic:
+        awss3:
+          s3uploader:
+              region: 'eu-central-1'
+              s3_bucket: 'databucket'
+              s3_prefix: 'metric'
+              s3_partition: 'minute'
+metadata:
+  logs:
+    statefulset:
+      extraEnvVars:
+        # Set env variables for awss3 exporter
+        - name: AWS_ACCESS_KEY_ID
+          value: "< YOUR AWS ACCESS KEY >"
+        - name: AWS_SECRET_ACCESS_KEY
+          value: "< YOUR AWS SECRET ACCESS KEY >"
+```
+
+### Send all logs to Sumo Logic and a part of it to other locations
+
+In case you want to send only part of logs to other location, you should use `sumologic.logs.otelcol.routing.table` in addition to `sumologic.logs.otelcol.extraExporters`.
+
+In the following example, all logs are going to be sent to Sumo Logic, and in addition only logs from the `production` namespace are going to be sent to AWS S3.
+
+```yaml
+sumologic:
+  logs:
+    otelcol:
+      extraExporters:
+        ## define `awss3/production` exporter
+        awss3/production:
+          s3uploader:
+              region: 'eu-central-1'
+              s3_bucket: 'databucket'
+              s3_prefix: 'metric'
+              s3_partition: 'minute'
+      routing:
+        table:
+          ## send all logs from `production` namespace to `awss3/production` exporter
+          ## as `useDefaultExporters` is set to `true` (default), all logs will be sent to `sumologic` as well
+          - exporter: awss3/production
+            statement: route() where resource.attributes["namespace"] == "production"
+metadata:
+  logs:
+    statefulset:
+      extraEnvVars:
+        # Set env variables for awss3 exporter
+        - name: AWS_ACCESS_KEY_ID
+          value: "< YOUR AWS ACCESS KEY >"
+        - name: AWS_SECRET_ACCESS_KEY
+          value: "< YOUR AWS SECRET ACCESS KEY >"
+```
+
+`sumologic.logs.otelcol.routing.table` is a list of maps, which consist of the keys `exporter` and `statement`.
+
+Logs are sent to `exporter` when `statement` is met. `exporter` should be defined in `sumologic.logs.otelcol.extraExporters`,
+or the default `sumologic` may be used. `statement` has to be written in
+[Open Telemetry Transformation Language](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/routingprocessor/README.md#tech-preview-opentelemetry-transformation-language-statements-as-routing-conditions).
+
+:::note
+Remember to use [attributes after translation to Sumo Logic schema](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/sumologicprocessor#attribute-translation). For xample, use `namespace` instead of `k8s.namespace.name`.
+:::
+
+:::note
+If a log matches multiple statements, it will be sent to all corresponding exporters.
+For example, the following routing table will send logs from the `production` namespace to `sumologic/production` and `awss3/production`:
```

Comment:
```suggestion
If a log matches multiple statements, it will be sent to all corresponding exporters. For example, the following routing table will send logs from the `production` namespace to `sumologic/production` and `awss3/production`:
```